tests/test_dependencies.py

```diff
@@ -326,6 +328,56 @@ def test_load_save_backward_compatibility(tmpdir, deps):
     assert deps == deps2
 
 
+@pytest.mark.parametrize("pandas_version", ["2.0.3", "2.1.4", "2.2.2"])
+def test_load_save_pandas_compatibility(pandas_version):
+    """Test pandas backward compatibility of pickle cache files.
+
+    Dataframes using pyarrow dtypes,
+    and stored as pickle files
+    might fail to load
+    if the used ``pandas`` version
+    does not match.
+
+    This tests documents,
```

Comment:
This sentence is conducive to understanding `documents` as the plural of  `document`. 
I wonder whether this can be rephrased.

Something simpler   "tests which pandas versions raise errors when loading deps from pickle files" or so?

Reply:
Good point, I changed to your suggestion.

Reply:
Resolving.